### PR TITLE
Missing createMany in TypeConverterResolverOpts for TypeScript

### PR DESCRIPTION
**Pull request by berrymat**
A small problem I happened to run across with the typescript definition for TypeConverterResolverOpts - it is missing the property for createMany.  It is present in the equivalent flow type definition.

### CODE REVIEW

File: src/composeWithMongoose.d.ts [hunk 1]
```diff
@@ -87,6 +87,11 @@ export type TypeConverterResolversOpts = {
     },
   createOne?:
     | false
+    | {
+      record?: RecordHelperArgsOpts | false,
+    },
+	createMany?:
+    | false
```
Comment (nodkz) [inline src/composeWithMongoose.d.ts]: OOps. Need to fix

**Comment by berrymat** [comment]
Is it the indentation that is wrong?  How can I run the same checks as the Travis CI locally before I submit an update?

**Comment by nodkz** [comment]
Just run 
`yarn test`
or
`npm run test`

**Comment by nodkz** [comment]
For checking only typescript, you may run following commands:
https://github.com/graphql-compose/graphql-compose-mongoose/blob/6dd16a198b6a1c557fdce3e90c2887a3a8ec0ed4/package.json#L99-L100

**Comment by berrymat** [comment]
Yes - using `yarn test` I could see the tslint failure!

**Comment by nodkz** [comment]
Thanks!

**Pull request merged**